### ISSUE TITLE
Can't send a test mention to webmention.io

### ISSUE THREAD

**Issue by AllienWorks** [mauricerenck/indieConnector#7]
Hello Maurice,

I'm playing with your plugin on my first ever Kirby site (so bear with me, I'm not a PHP pro either :sweat_smile:).

I've installed Komments (already working) and now trying to add the indieConnector.

- both plugins installed
- secret added to `config.php`
- when visiting `https://<domain.tld>/indieconnector/send-test-mention/<secret>` I do see the "Sent! You should be able to configure your webmention.io hook now." message
- ..but nothing shows up on my https://webmention.io/settings/webhooks dashboard
- also when I visit `/panel/webmentions`, I get this error: `this.$store is undefined` (not sure if it's related or normal at this point of installation?)

Any ideas what might be the issue?
I'll happily send all the requested logs and whatnot :)

Versions below:

- PHP 8.2.7
- Kirby 3.9.7
- indieConnector 1.8.1
- komments 1.12.1

**Comment by mauricerenck** [comment]
Hey,

It's the domain you're testing this from, reachable from outside or is it a local domain? 

If it is public make sure to add the webmention endpoint to your HTML header using the snippet. You could also try to send a when to yourself by linking from one page to another, but make sure sending mentions to yourself isn't disabled.

The error in the panel may be the result of a missing database. Did you set up sqlite? If so, is the database file created?

**Comment by AllienWorks** [comment]
Hello, sorry for late reply.

> It's the domain you're testing this from, reachable from outside or is it a local domain?

This has been observed on a live domain already (so publicly reachable).

> If it is public make sure to add the webmention endpoint to your HTML header using the snippet.

These are present in `<head>`, just double-checked:

```html
<!-- Webmentions -->
<link rel="pingback" href="https://webmention.io/<domain.tld>/xmlrpc" />
<link rel="webmention" href="https://webmention.io/<domain.tld>/webmention" />
```

> You could also try to send a when to yourself by linking from one page to another, but make sure sending mentions to yourself isn't disabled.

I have not tried this yet, will do!

> The error in the panel may be the result of a missing database. Did you set up sqlite? If so, is the database file created?

I initially tried (had it in config), but it broke the site due to some migrations error (all I remember now). So I turned it off since I don't really need it now. (My host might not support it.)

**Comment by mauricerenck** [comment]
Would you mind sending me the url? I could then try send a webmention and maybe get a clue what's going on. You can send it to me directly if you don't want to post it here.

**Comment by AllienWorks** [comment]
Sorry, I haven't had much time to play with this. I tried this (also might be url you're interested in): https://webmention.io/allien.work/webmention - to send a webmention from one of my pages to another. I got a `Internal Server Error` so I wonder if screwed it up or whether it might be a problem on webmention.io's side?

**Comment by mauricerenck** [comment]
Unfortunately it's pretty hard to test things like this, because you cannot look into the systems. I tried to send you a webmention and at least didn't get an error. Please have a look into webmention.io and check if you can see anything there.

**Comment by AllienWorks** [comment]
Yeah I'm sure, I wasn't exactly expecting you to solve it completely by your own. (Maybe just nudge me in the right direction.)

..and behold! I can now see your mention on the Webmention.io! (also one of mines from earlier today it seems). So let's see how will it work from now on. Let me play with it and report back to you!

**Comment by AllienWorks** [comment]
So I did the rest of the configuration (setting up the hook on Webmention.io). So far nothing changed – is that the expected behaviour?

- Your test mention is there: https://webmention.io/api/mentions.html?token=N5dF6-qsAyhoXivXYlqV5w
- But nothing shows up on the article: https://allien.work/en/articles/mask-email-with-aliasases

Will only the future webmentions appear in the Komments? I did check and I have not specified any webmention-related config for Komments (and all of those defaults to `true` - enabled support, auto-publish..).

Sorry for newbie questions..
